### PR TITLE
CLI Release 1.11.0 rc.0 changes to develop

### PR DESCRIPTION
**Pull request by wshands**

### CODE REVIEW

**Review by denis-yuen — COMMENTED** [review]
Changes looked reasonable but there looked like relevant build issues to resolve

**Review by coverbeck — COMMENTED** [review]
Was here, seconding Denis' comment on build issues.

**Review by coverbeck — COMMENTED** [review]
Shouldn't there be a release/1.11.0 branch, with a PR against that?

**Comment by coverbeck** [comment]
> Shouldn't there be a release/1.11.0 branch, with a PR against that?

On further reflection, I guess if nothing has changed on develop, it will work.

File: dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java [hunk 1]
```diff
@@ -303,11 +303,11 @@ private Workflow getDockstoreWorkflowByPath(String path) {
         // simply getting published descriptors does not require permissions
         Workflow workflow = null;
         try {
-            workflow = workflowsApi.getPublishedWorkflowByPath(path, null, false, null);
+            workflow = workflowsApi.getPublishedWorkflowByPath(path, "versions", false, null);
```
Comment (coverbeck) [inline dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java:306]: This means that the 1.10 client will break against the 1.11 web service, doesn't it? I don't think there's much to do about it at this point.

**Review by coverbeck — APPROVED** [review]

**Review by garyluu — APPROVED** [review]

File: dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java (hunk 1 above)
Comment (wshands) [reply dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java:306]: Yes

**Review by NatalieEO — APPROVED** [review]

**Review by Richard-Hansen — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**